css/qmwp-search-correlations.css

```diff
@@ -288,23 +288,37 @@
     margin-bottom: 10px;
 }
 
+#searchResultList a {
+    color: rgba(66, 133, 244, 1);
+}
+
 #searchResultList h4 {
     margin-top: 10px;
     margin-bottom: 5px;
+    font-weight: 700;
 }
 
 #searchResultList .search-result {
+    margin: 7px 0;
+}
+
+.details-and-controls {
     display: flex;
     align-items: center;
-    margin: 7px 0;
 }
 
-.vote-thumb {
+.controls > * {
+    font-size: 32px;
     cursor: pointer;
+    opacity: 0.5;
+}
+
+.controls > *:hover {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The universal selector (*) is known to be slow.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1239003805/issues/source?bid=2836188&fileBranchId=2935424#l316)